internal/confmapprovider/discovery/bundle/bundle.d/receivers/mongodb.discovery.yaml

```diff
@@ -0,0 +1,51 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+mongodb:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)mongo"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)mongo" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)mongo"
+  config:
+    default:
+      username: splunk.discovery.default
+      password: splunk.discovery.default
+      tls:
+        insecure: true
+        insecure_skip_verify: true
+  status:
+    metrics:
+      - status: successful
+        strict: mongodb.database.count
+        log_record:
+          body: mongodb receiver is working!
+    statements:
+      - status: failed
+        regexp: 'connect: network is unreachable'
+        log_record:
+          append_pattern: true
+          body: The container cannot be reached by the Collector. Make sure they're in the same network.
+      - status: failed
+        regexp: 'connect: connection refused'
+        log_record:
+          append_pattern: true
+          body: The container is refusing mongodb connections.
+      - status: partial
+        regexp: '.* unable to authenticate using mechanism .*'
+        log_record:
+          append_pattern: true
+          body: >-
+              Please ensure your user credentials are correctly specified with
```

Comment:
Should we specify which role we need credentials for, given that we mention `admin` in the other rule

Reply:
Admin is not necessary to perform these action but the 'user' needs to have readWrite access to perform these actions.